widget_driver/example/lib/widgets/home_page/tabs/home_page_tabs.dart

```diff
@@ -1,10 +1,10 @@
 import 'package:flutter/material.dart';
 
 class HomePageAppTabs {
-  final List<AppTabType> tabs = [AppTabType.consumption, AppTabType.community];
+  final List<AppTabType> tabs = [AppTabType.consumption, AppTabType.community, AppTabType.playground];
```

Comment:
I guess that if you merge in latest "origin/master" into this PR, then this change goes away.. since this is already on master